### DISCUSSION TITLE
Images not loading on the activity screen.

### DISCUSSION THREAD

**Discussion by 4y55** [nextcloud/all-in-one#1359] [Questions] [answered]
Images not loading on the activity screen.
It has some relationship with the collaborator. The images go back to normal when I disable the collabora app.
How to solve?

![erro2](https://user-images.githubusercontent.com/50604378/199984382-2de112c7-f662-4cb4-bf8a-0b876c7da30f.jpg)
![erro1](https://user-images.githubusercontent.com/50604378/199984387-b91ff0ab-40f7-4f91-8f71-bb2ce1262228.jpg)

**Comment by szaimen** [comment, marked as answer]
Yes, see https://github.com/nextcloud/activity/issues/760
